### PR TITLE
use daemon_name instead of service_name if daemon_name differs

### PR DESCRIPTION
**Pull request by hex2a**
this PR implements behavior as described in services_disabled.csv:
`service_name, package_name, daemon_name (as recognized by chkconfig / systemd. To be used when daemon_name differs from service_name)`

If daemon_name is given, it will be used as service name for chkconfig/systemd to check whether the service is running, and for remediations.

An example would be sshd/ntpd for Ubuntu/Debian, which runs as ssh/ntp service.

### CODE REVIEW

**Comment by mpreisler** [comment]
NACK.

create_service_enabled.py also needs to be changed.

File: shared/templates/create_services_enabled.py [hunk 1]
```diff
@@ -20,23 +20,35 @@
 class ServiceEnabledGenerator(FilesGenerator):
 
     def generate(self, target, serviceinfo):
-        # get the items out of the list
-        servicename, packagename = serviceinfo
+        try:
+            # get the items out of the list
+            servicename, packagename, daemonname = serviceinfo
+        except ValueError as e:
+            print "\tEntry: %s\n" % serviceinfo
```
Comment (mpreisler) [inline shared/templates/create_services_enabled.py]: use python3 compatible syntax

Comment (hex2a) [inline shared/templates/create_services_enabled.py]: same code as in [create_services_disabled.py](https://github.com/hex2a/scap-security-guide/blob/ebd2a1e8f85c34efee174b6dee46a78f03207403/shared/templates/create_services_disabled.py#L24), interpreter is explicitly set to python2

Comment (mpreisler) [inline shared/templates/create_services_enabled.py]: yes but it doesn't hurt to use python3 compatible syntax even in python2. there is no reason to use old obsolete syntax.

Comment (hex2a) [inline shared/templates/create_services_enabled.py]: fixed, also fixed some pylint indentation errors

Comment (hex2a) [inline shared/templates/create_services_enabled.py]: @mpreisler could you review again? :)

**Comment by mpreisler** [comment]
ACK

**Pull request merged**